code/modules/events/antag_ghost_respawn.dm

```diff
@@ -44,6 +44,12 @@
 				return
 
 			src.antagonist_type = pick(list("Blob", "Hunter", "Werewolf", "Wizard", "Wraith", "Wrestler", "Wrestler_Doodle", "Vampire", "Changeling", "Flockmind"))
+			for(var/mob/wraith/W in ROLE_WRAITH)
```

Comment:
also this loop condition does not look right. I think you're iterating over the wrong thing

Reply:
I'm trying to have it check that out of the current wraiths around, which ones have died less than twice. I'm unsure if there's something else that checks if a wraith is dead-dead.

Reply:
I think I see the problem, gonna replace the ROLE_WRAITH with the list for selected traitors.